Sources/TuistKit/Commands/Cloud/CloudNextCommand.swift

```diff
@@ -0,0 +1,16 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+
+struct CloudNextCommand: ParsableCommand {
```

Comment:
What if instead of exposing a new namespace, we use a compile-time macro that we control via environment variables to add/remove commands from the existing `cloud` namespace? Not sure if that's possible at compile-time, otherwise this is fine.

Reply:
As discussed, we will iterate with the production backend, so there's no need to put these features behind a feature flag.